devops/jobs/CreateSandboxSSHAccess.groovy

```diff
@@ -70,6 +70,8 @@ class CreateSandboxSSHAccess {
         sshAgent('sandbox-ssh-keys')
       }
 
+      authenticationToken(extraVars.get('JENKINS_SANDBOX_JOB_KEY', 'S@nD80%'))
+
```

Comment:
The actual key was created using LastPass's Key generator. I added a default key just in case

Reply:
Dont add default passwords because if you forget to set them then they are known, this repo is public as well. It would be better to blow up if it didn't exist, or skip it if it didn't exist.

Reply:
@coryleeio Done 👍